### PR TITLE
SQL versioning for Az

### PR DESCRIPTION
**Pull request by maddieclayton**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
az sign job: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/218/

File: src/ResourceManager/Sql/Commands.Sql/Az.Sql.psd1 [hunk 1]
```diff
@@ -286,10 +294,15 @@ PrivateData = @{
...
+        ReleaseNotes = '* New Cmdlets for Management.Sql to allow customers to add TDE keys and set TDE protector for managed instances
+   - Add-AzSqlManagedInstanceKeyVaultKey
+   - Get-AzSqlManagedInstanceKeyVaultKey
+   - Remove-AzSqlManagedInstanceKeyVaultKey
+   - Get-AzSqlManagedInstanceTransparentDataEncryptionProtector
+   - Set-AzSqlManagedInstanceTransparentDataEncryptionProtector'
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```
Comment (cormacpayne) [inline src/ResourceManager/Sql/Commands.Sql/Az.Sql.psd1]: @maddieclayton for `Az`, we're going with the semantic version pattern of `0.x.x` being preview (or unstable), and only roll out `-preview` versions for stable modules, so you can revert this change

**Comment by maddieclayton** [comment]
netsign job here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/221/
